cytosnake/utils/cyto_paths.py

```diff
@@ -153,6 +153,21 @@ def get_config_dir_path() -> Path:
     return config_path
 
 
+def get_cytosnake_config_path() -> pathlib.Path:
+    """Returns absolute path to CytoSnake's general config file .
+
+    Returns
+    -------
+    pathlib.Path
+        path to cytosnake general config file
+    """
+    config_dir_path = get_config_dir_path() / "configuration.yaml"
```

Comment:
consider making `configuration.yaml` a function default argument. (If this is extra busy work, it is totally ok to tell me that you think this fix is low priority and not do it 😄 )

Reply:
Are you suggesting something like this? 
```python
def get_cytosnake_config_path(config_path: Optional[str] = "configuration.yaml") -> pathlib.Path:
     """Returns absolute path to CytoSnake's general config file .
     Returns
     -------
     pathlib.Path
         path to cytosnake general config file
     """
     config_dir_path = get_config_dir_path() / config_path
```